client/src/Components/local/atoms/core-mapping/index.tsx

```diff
@@ -103,16 +112,23 @@ const CoreMapping: React.FC<PropTypes> = ({ messages, channel, playerForce, play
         timestamp: timestamp,
         turnNumber: 1
       }
-      const newMessage: MappingMessage = {
-        details, 
-        featureCollection,
-        messageType: CUSTOM_MESSAGE,
-        _id: timestamp
+
```

Comment:
Aah, @lebaphi - there is another bit of logic we can use here.  If there **isn't** a `lastMessage` then we send the whole `newFeatureCollection` as a `MAPPING_MESSAGE`.

At some point in the future we will need to consider when we send these whole messages.  But, when do come up with something, we will just have to clear `lastMessage` - and know that the full `featureCollection` will be sent.